client/src/use/useTrackFilters.spec.ts

```diff
@@ -0,0 +1,53 @@
+/// <reference types="jest" />
+import Vue from 'vue';
+import CompositionApi from '@vue/composition-api';
+import Track, { Feature } from '../track';
+import useTrackFilters from './useTrackFilters';
+import useTrackStore from './useTrackStore';
+
+Vue.use(CompositionApi);
+
+const removeTrack = () => null;
+const markChangesPending = () => null;
+
+/**
+ * Tracks need to be initialized with features
+ * in order to broadcast notifications
+ */
+const features: Feature[] = [
+  {
+    frame: 0,
+    bounds: [1, 2, 3, 4],
+    head: [1, 2],
+    keyframe: true,
+  },
+];
+
+function makeTrackStore() {
+  const t0 = new Track(0, {
+    confidencePairs: [['foo', 0.5], ['bar', 0.4]],
+    features,
+  });
+  const t1 = new Track(2, {
+    confidencePairs: [['foo', 0.9], ['baz', 0.2]],
+    features,
+  });
+  const t2 = new Track(200, {
+    confidencePairs: [['bar', 1], ['baz', 0.8]],
+    features,
+  });
+  const ts = useTrackStore({ markChangesPending: () => null });
+  ts.insertTrack(t0);
+  ts.insertTrack(t1);
+  ts.insertTrack(t2);
+  return ts;
+}
+
+describe('useTrackFilters', () => {
+  it('renames tracks', () => {
+    const { sortedTracks } = makeTrackStore();
+    const tf = useTrackFilters({ sortedTracks, removeTrack, markChangesPending });
+    tf.updateTypeName({ currentType: 'foo', newType: 'baz' });
+    expect(tf.allTypes.value).toEqual(['baz', 'bar']);
+  });
+});
```

Comment:
feel free to ignore this.  Test for renaming type to existing type and making sure that the counts are additive.  This currently works but a test may be nice.

Reply:
I'm not sure what you mean by making sure the counts are addative.

Reply:
`foo` has 2 tracks, `bar` has 2 tracks, rename `bar` to `foo`, ensure that `bar` now has 4 tracks?